exporters/prometheus/config.go

```diff
@@ -95,6 +93,20 @@ func WithoutTargetInfo() Option {
 	})
 }
 
+// WithTranslationStrategy provides a standardized way to define how metric and
+// label names should be handled during translation to Prometheus format. The
+// recommended approach is to use either UnderscoreEscapingWithSuffixes for full
+// Prometheus-style compatibility (the default), or NoTranslation for Otel-style
+// names. For convenience, this option overrides the WithCounterSuffixes option.
```

Comment:
I would expect WithoutCounterSuffixes to override the translation strategy, since it is an intentional addition to the config--if we keep this config option at all. If a user sets: `WithTranslationStrategy(UnderscoreEscapingWithSuffixes), WithoutCounterSuffixes()`, I would expect it to only include unit suffixes, and not counter suffixes.

Reply:
Hmmmm, wouldn't it be clearer to not allow both at the same time? We could add a warning log saying that WithoutCounterSuffixes won't do anything if the translation strategy is set.

Just throwing some ideas